[2024-07-25T16:37:09.898+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-25T16:37:09.930+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_format_dag_v3.task1 scheduled__2024-07-09T03:00:00+00:00 [queued]>
[2024-07-25T16:37:09.937+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_format_dag_v3.task1 scheduled__2024-07-09T03:00:00+00:00 [queued]>
[2024-07-25T16:37:09.938+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-25T16:37:09.953+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task1> on 2024-07-09 03:00:00+00:00
[2024-07-25T16:37:09.961+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'cron_format_dag_v3', 'task1', 'scheduled__2024-07-09T03:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/cron_dag.py', '--cfg-path', '/tmp/tmpqs8z9upp']
[2024-07-25T16:37:09.962+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=138) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-25T16:37:09.967+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask task1
[2024-07-25T16:37:09.970+0000] {standard_task_runner.py:64} INFO - Started process 145 to run task
[2024-07-25T16:37:10.038+0000] {task_command.py:426} INFO - Running <TaskInstance: cron_format_dag_v3.task1 scheduled__2024-07-09T03:00:00+00:00 [running]> on host 7c5f22b81ce4
[2024-07-25T16:37:10.214+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Pattakit' AIRFLOW_CTX_DAG_ID='cron_format_dag_v3' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-07-09T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-09T03:00:00+00:00'
[2024-07-25T16:37:10.216+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-25T16:37:10.331+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-25T16:37:10.343+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression']
[2024-07-25T16:37:10.373+0000] {subprocess.py:86} INFO - Output:
[2024-07-25T16:37:10.374+0000] {subprocess.py:93} INFO - dag with cron expression
[2024-07-25T16:37:10.375+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-25T16:37:10.380+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-25T16:37:10.402+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=cron_format_dag_v3, task_id=task1, run_id=scheduled__2024-07-09T03:00:00+00:00, execution_date=20240709T030000, start_date=20240725T163709, end_date=20240725T163710
[2024-07-25T16:37:10.446+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-25T16:37:10.470+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-25T16:37:10.471+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
